### DISCUSSION TITLE
Question about dependecy

### DISCUSSION THREAD

**Discussion by f3d0ss** [PaulRBerg/hardhat-template#95] [Q&A] [answered]
Why do you import those dependency:
https://github.com/paulrberg/solidity-template/blob/315cf7a478e8bf149a13a4d42a4d81038d3495b2/package.json#L13-L17
When you have this one:
https://github.com/paulrberg/solidity-template/blob/315cf7a478e8bf149a13a4d42a4d81038d3495b2/package.json#L36
Shouldn't those be already included?
I'm asking because I'm kind of new to those things

**Comment by PaulRBerg** [comment, marked as answer]
The umbrella package `ethers` is needed because it's a peer dependency of many other packages, e.g. `@nomiclabs/hardhat-ethers`.
